x-pack/plugins/ml/public/application/components/data_grid/data_grid.tsx

```diff
@@ -261,6 +263,16 @@ export const DataGrid: FC<Props> = memo(
       }
     }
 
+    const onMutation = () => {
+      const els = document.querySelectorAll('.euiDataGrid__virtualized');
```

Comment:
with react we should avoid using `document`, maybe `mutationRef` will work out?

Reply:
Fixed in 081ae1b. I put another ref on the outer `div` element because `mutationRef` is only available on the inner callback of `EuiMutationObserver`.